Getting-Started-with-InnerSource/ja.md

```diff
@@ -218,27 +223,25 @@ Node.js は明らかに勝っていました。
 
 PayPal が Node.js を採用するという発表は、多くのプログラマーを、特に C++ から Java への移行が長くて大変だったことを記憶しているプログラマーを心配させました。
 しかし、PayPal は、Node.js への移行がまったく違うものであることを示しました。
-PayPal は、移行トレーニングにわずか2日間のスケジュールを組みました。
-Harrell によると、参加者はすぐに、自分たちが新しく、活気があり、エキサイティングな世界に入ったことを実感したといいます。
+PayPal は、移行トレーニングにわずか 2 日間のスケジュールを組みました。
+Harrell によると、トレーニングの参加者は、新しく活気があってエキサイティングな世界に入ったことをすぐに実感したそうです。
 さらに、JavaScript の分野のリーダーである Douglas Crockford をトレーニングのスタッフとして迎えられたことも幸運でした(Crockford は、非常に人気の高い書籍 _[JavaScript: The Good Parts](http://shop.oreilly.com/product/9780596517748.do)_ の著者であり、オライリーの _[JavaScript Master Classビデオ](http://shop.oreilly.com/product/9780596809614.do)_ の制作者でもあります)。
 
 現在、PayPal の新機能開発チームの大半は Java を利用していますが、その一方で、Node.js チームが開拓したインナーソースのプラクティスを採用しているチームが存在します。
 
-### OSS との関わり
+### OSS との調和
 
 Node.js に移行するためのトレーニングは、多くの教材がオンラインで利用できたことと、Node.js は前回の Java への移行に比べてカスタマイズの必要性が少なかったことから、簡単に短時間で実施できました。
 また、スタッフが自分自身の学習プロセスを継続できるよう準備するには、基本的に2日間のトレーニングで十分でした。
 
 Harrell によると、基本的な情報は社内のメーリングリストでシニアプログラマーに尋ねるのではなく、単に StackOverflow などのサイト (あるいは検索エンジンなど) で検索すれば良いということを PayPal のプログラマーへ伝えるのに時間がかかったそうです。
 しかし、やがて情報の問い合わせは OSS コミュニティへのひたむきな取り組みにまで広がりました。
 
-多くの企業は OSS の技術が単に高品質で無料であるということで魅力的を感じる一方、採用することで OSS コミュニティへの継続的な参入につながると考えています。
+多くの企業は、OSS という技術が純粋に高品質で無償であることを魅力とみなしており、それをきっかけにした OSS の採用はコミュニティへの継続的な参入につながると考えています。
```

Comment:
> 「一方」という言葉の持つ対比感があまりない気がします。「採用することで」→「採用することは」のほうが自然かもしれません

とのことです。
原文は `Most companies find that adopting open source technologies—which may originally be attractive simply because they’re high quality and free of charge—leads to a sustained entry into the communities that produce the technologies.` であり、「一方」 に相当する表現はありません。